packages/react/select/src/Select.tsx

```diff
@@ -717,7 +718,7 @@ const SelectContentImpl = React.forwardRef<SelectContentImplElement, SelectConte
         searchRef={searchRef}
       >
         <Portal>
-          <RemoveScroll>
+          <RemoveScroll as={Slot}>
```

Comment:
I realised here also `Select` is missing the pinchZoom prop, I'll handle this separately.